solidity/contracts/GasRouter.sol

```diff
@@ -50,9 +50,12 @@ abstract contract GasRouter is Router {
                 _destinationDomain,
                 _mustHaveRemoteRouter(_destinationDomain),
                 "",
-                IGPMetadata.formatMetadata(
+                GlobalHookMetadata.formatMetadata(
+                    1,
```

Comment:
I think the idea was that the version is defined by the implementation but the user shouldn't have to know about versions
so `formatMetadata` would prepend whatever version